draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -526,7 +526,7 @@ The UUID format is 16 octets (128 bits); the variant bits in conjunction with th
 bits described in the next sections in determine finer structure. While discussing UUID formats and layout, bit definitions start at 0 and end at 127 while octet definitions start at 0 and end at 15.
 
 UUIDs MAY be represented as binary data or integers.
-When in use with URNs or applications, any given UUID SHOULD
+When in use with URNs or as text in applications, any given UUID SHOULD
```

Comment:
As mentioned elsewhere URNs are just one aspect of UUIDs, so I think this helps make it more clear.